### DISCUSSION TITLE
Extra 4px in Viz Picker?

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3742] [General] [answered]
Hey team - I noticed that there's an extra 4px to the right of the RadioDot in the visual picker. It looks intentional in the radio button component in order to allow 8px in between the input and label, but wondering if it's intentional for visual picker, since that leaves about 20px in between the RadioDot and text:
 
![Screenshot 2024-01-29 at 11 47 11 AM](https://github.com/twilio-labs/paste/assets/87141551/13d2ee8e-a8f2-46bc-b4ac-169c558490bc)

I'm working on refining the delivery overview designs, and need all the pixels I can get, thanks!

**Comment by serifluous** [comment]
Hey @ceyerkes ! Looks like this was an oversight when we finalized the Figma + code components from the design explorations.

I've got a PR up for the fix here: #3743 and have the Figma library out for review too.

**Reply by ceyerkes** [reply]
thanks @serifluous!! I reduced padding to 8px on either side of the RadioDot in order to squeeze this overview pipeline as much as possible:

![Screenshot 2024-01-29 at 3 34 10 PM](https://github.com/twilio-labs/paste/assets/87141551/972ba464-328f-4694-ab8c-1fa73f30a3b3)

Hopefully that doesn't look too cramped, open to feedback!

**Reply by serifluous** [reply, marked as answer]
You won't be able to change the padding in the component unfortunately :( unless we reduce it down for everyone.

**Reply by lnamba** [reply]
Hi @serifluous! I was working on trying to change the padding to 8px on the visual picker, but ran into issues and found this discussion. It seems this is a limitation of the component currently. Are there any plans to add a padding prop or something else to adjust this at all?

**Reply by serifluous** [reply]
Hey @lnamba ! We don't have any plans to do that right now—you'll have to readjust the layout of each card, or build something custom if you're finding the horizontal space too tight.
